### DISCUSSION TITLE
Support for reading/writing DNG files

### DISCUSSION THREAD

**Discussion by dejanberic** [SixLabors/ImageSharp#2428] [Ideas]
I know that there is Adobe DNG SDK but that is written in C++ and I do not want to deal with that.
Ideally I am looking for a C# library with support for reading DNG files, and writing DNG files.
This would be suited for cross platform photo editors which want to support raw files from cameras.
I started porting Adobe DNG SDK to C# but I was very naïve thinking it would be easy :)

Is there a plan to support DNG format or no?
If this library will never support it, does anyone know of any other C# library which supports reading and writing DNG files?

**Comment by antonfirsov** [comment]
If there is a proposal for providing support by community contribution we may consider it, otherwise I don't think it's a use-case we would invest into.

@Webreaper any opinions on this format?

**Comment by Webreaper** [comment]
No opinions from me; I'm really only interested in reading/decoding images - the only time I really write files is as JPEG.
